app/Dockerfile

```diff
@@ -32,10 +32,16 @@ RUN apk add --update \
     py-pip \
     python \
     python-dev \
+    ruby \
+    ruby-bundler \
+    ruby-dev \
```

Comment:
Thanks for asking about that. It made me question which packages I needed and prove that I needed what I'm adding.